controller/src/enhancements/bomb.rs

```diff
@@ -110,6 +196,39 @@ impl Enhancement for BombInfoIndicator {
             PlantedC4State::NotPlanted => unreachable!(),
         }
 
+        if let PlantedC4State::Active { .. } = &bomb_state.state {
+            if let Some(pawn_info) = self.get_view_pawn_info(

```

Comment:
afaik this is not needed as `LocalCameraControllerTarget` will resolve the actually spectated entity id (e.g. the player pawn) already.